### ISSUE TITLE
Make it possible to generate a time series from a Time object

### ISSUE THREAD

**Issue by eteq** [astropy/astropy#8596]
This is a follow-on idea from #8540, particularly https://github.com/astropy/astropy/pull/8540#issuecomment-482948959 and https://github.com/astropy/astropy/pull/8540#issuecomment-484507538 .  The idea is to add another way of creating `TimeSeries` by doing ``time.series()`` (where `time` is a `Time` object).

TBD: whether keywords should be allowed to have more control of `TimeSeries` creation - e.g. adding more data columns.

(I'm initially milestoning it for 4.0 but it could get pulled up to 3.2 as a quick follow-on to #8540 if I find time and someone can review it by tomorrow.)

**Comment by astrofrog** [comment]
I'm not fully convinced we really need this - after all, the Zen of Python says

*There should be one-- and preferably only one --obvious way to do it.*

What would be the benefit of being able to do ``Time.series``? Would it be for use in contexts other than the ``TimeSeries`` class?

Would it make more sense to focus on making sure that e.g. ``np.linspace`` works for ``Time`` as @mhvk mentioned in #8540?

**Comment by mhvk** [comment]
I agree that if we have `np.linspace` work, there is little reason for also having a method. Adjusting the title to more generally have a nice way to generate time series.

**Comment by eteq** [comment]
Hmm.  Point well taken after I explored a bit more and realized that `TimeSeries(time=timeobj)` would do the same thing.  My original motivation would that it would be more complicated than that.

The more minor motivations are
1. That way I don't need to import `TimeSeries` to get a series from a `Time` object
2. More tab-discoverability

And honestly I don't think those out-weigh @astrofrog's quite right invocation of the Zen...

So perhaps we should morph this into the (conveniently not feature-freeze-requiring...) "document clearly that TimeSeries(time=timeobj) can be used? (I don't see it until deep into the creating docs, and even there it's not quite that simple).

**Comment by astrofrog** [comment]
I can try and clarify the docs next week! Indeed you can just do ``TimeSeries(time=timeobj, data=...)``

**Comment by pllim** [comment]
From the discussion, this is not gonna happen by 4.0 if it is gonna happen at all, so I am removing the milestone.
